.travis.yml

```diff
@@ -29,45 +29,24 @@ before_script:
 script:
   - ./gradlew build --stacktrace
 
-# Deploy build artifacts; Travis does not invoke this step for pull request builds
+# Deploy build artifacts. Travis does not invoke this step for pull request builds
 deploy:
-  # Publish SNAPSHOT artifacts to oss.jfrog.org
-#  - provider: script
-#    # OJO publishing not available yet
-#     script: ./gradlew worldwind:artifactoryPublish --stacktrace
-#    skip_cleanup: true
-#    on:
-#      branch: develop
   # Publish release artifacts to Bintray/JCenter
   - provider: script
     script: ./gradlew worldwind:bintrayUpload --stacktrace
     skip_cleanup: true
     on:
       tags: true
-  # Publish release documentation to GitHub pages
+  # Create CHANGELOG.md in the current directory
   - provider: script
-    script: ./travis/ghpages.sh
+    script: 'source ./travis/changelog.sh && changelog "${TRAVIS_TAG}" "https://api.github.com/repos/${TRAVIS_REPO_SLUG}" >> CHANGELOG.md'
```

Comment:
The travis environment was not able to find 'source'. After some searching, I found the following Stack Overflow discussion: https://stackoverflow.com/questions/13702425/source-command-not-found-in-sh-shell

I attempted to specify '.' as suggested by the SO discussion but received a new error: https://travis-ci.org/NASAWorldWind/WorldWindAndroid/builds/242257638

I'm in the process of downloading Ubuntu 12.0.4.5 and running on a VM to see if the issue can be replicated locally.